ide/coqide/coqOps.ml

```diff
@@ -40,13 +40,16 @@ module SentenceId : sig
     mutable flags : flag list;
     mutable tooltips : (int * int * string) list;
     start_offset : int;
+    (** Original start offset of the sentence, to be compared when moved *)
+    start_line : int;
+    (** Original start line of the sentence, to be compared when moved *)
```

Comment:
To clarify semantics, "Original" here means "location Coq thinks the sentence is at." 

The `start` / `stop` marks do contain the buffer real location of the sentence, so `offset_compensation` below can be used to apply / track the delta between the two.